budproj/business
# Fix/key result type

## 🎢 Motivation

update key result's polarity when update key result 

## 🔧 Solution

implements type field to key result input form and implements logic to alter the value

## 🚨  Testing

1. edit a key result
2. alter initial value
3. alter goal
4. check if progress inverts 

## 🃏 Task Card

Link to issue on Jira

- [`BUD22-123`](https://www.notion.so/budops/Bug-ao-trocar-a-polaridade-do-kr-1da764a4d5294cd1972dbd5a08872e39?pvs=4)

## 🔗 Related PRs

This PR is related to some other PRs in different services, they are:

- [`project#PR_NUMBER`](https://)

## 🍩 Validation

Who tested this PR?

### Canary


### Dev
